### ISSUE TITLE
[Accessibility] python repl content is not parsable in the a11y terminal buffer when Powershell is used

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#174797]
CC @meganrogge

* Note: the following issue is not found with `cmd Command Prompt` terminal profile. I suspect there is something that has to do with integrated shell.

## Reproducible Steps

1. Load NVDA.

1. Run `Terminal: Create New Terminal (With Profile)`.

1. Choose `terminal-powershell`

1. In Powershell, type `python` to open python repl.

1. Type the following in python shell and hit enter:

``` python
1+2
```

1. See if you can read ipython content in a11y terminal buffer.

## Current Behavior

``` bash
No terminal content available for this session. Run some commands to create content.
```

## Expected Behavior

python content should be accessible in the terminal buffer.

## System Info

Version: 1.76.0-insider (system setup)
Commit: 96a795cc0b1cd62f9ff66c347637f917c6dde3da
Date: 2023-02-17T22:22:55.538Z
Electron: 19.1.9
Chromium: 102.0.5005.194
Node.js: 16.14.2
V8: 10.2.154.23-electron.0
OS: Windows_NT x64 10.0.22621
Sandboxed: Yes

**Comment by meganrogge** [comment]
This one's tricky because we don't trap the output of sub-shells ATM. Perhaps we should have a setting that lets the user decide whether or not we show the full contents of the buffer vs the shell integration contents

**Comment by jooyoungseo** [comment]
Yes, having that user setting would minimize this confusion.

**Comment by meganrogge** [comment]
As we've discussed the other way to work around this is by disabling shell integration. Perhaps we should just present the whole buffer when a screen reader is detected?

Do you think it would be useful to have cwd, the command, and output via shell integration or should we just provide the whole buffer @jooyoungseo ?

there's also the idea of using symbols / an outline view eventually #174079

**Comment by jooyoungseo** [comment]
@meganrogge -- Yes, the whole output would be preferred. I have detailed some more ideas about the terminal symbol providers in my response to #174079.

**Comment by meganrogge** [comment]
We are planning to present all of the contents of the terminal buffer going forward, so this won't be an issue
